templates/hosts.cfg

```diff
@@ -2,26 +2,41 @@ define host{
   name                            base-host             ; The name of this host template
   notifications_enabled           1                     ; Host notifications are enabled
   event_handler_enabled           1                     ; Host event handler is enabled
-  flap_detection_enabled          0                     ; Flap detection is enabled
+  flap_detection_enabled          0                     ; Flap detection is disabled
   process_perf_data               1                     ; Process performance data
   retain_status_information       1                     ; Retain status information across program restarts
   retain_nonstatus_information    1                     ; Retain non-status information across program restarts
   notification_period             24x7                  ; Send host notifications at any time
   notification_options            d,u,r                 ; Only send notifications for specific host states
   contact_groups                  admins                ; Notifications get sent to the admins by default
   register                        0                     ; DONT REGISTER THIS DEFINITION - ITS NOT A REAL HOST, JUST A TEMPLATE!
-  check_interval                  1                     ; Actively check the host every 5 minutes
+  check_interval                  1                     ; Actively check the host every minute
   retry_interval                  1                     ; Schedule host check retries at 1 minute intervals
-  max_check_attempts              3                     ; Check each Linux host 10 times (max)
+  max_check_attempts              3                     ; Check each Linux host 3 times (max) before alerting
   check_command                   check-host-alive      ; Default command to check Linux hosts
-  notification_interval           30                    ; Resend notifications every 2 hours
+  notification_interval           30                    ; Resend notifications every 30 minutes
 }
 
 {% for host in nagios_hosts %}
 define host{
-  use         base-host
-  host_name   {{host.name}}
-  alias       {{host.address}}
-  address     {{host.address}}
+  use                   base-host
+  host_name             {{host.name}}
+  alias                 {{host.address}}
+  address               {{host.address}}
+{% if host.contact_groups|default(false) %}
+  contact_groups        {{host.contact_groups}}
+{% endif %}
+{% if host.check_interval|default(false) %}
+  check_interval        {{host.check_interval}}
+{% endif %}
+{% if host.retry_interval|default(false) %}
```

Comment:
We should probably set these values in `/defaults/main.yml`, rather than the inline logic; a maintenance  task when we do this, we should move `/vars/main.yml` to `/defaults/main.yml` (this is the convention, which I didn't know at the time).